financialaid/api_test.py

```diff
@@ -35,7 +35,7 @@
 from roles.roles import Staff, Instructor
 
 
-def create_program():
+def create_program(create_tiers=True):
```

Comment:
since not all tests need these tier programs, i think it makes more sense to split this into `create_program` and `create_program_with_tiers`. tests that only need the program can use `program = create_program()`, and tests that need both can use `program, tier_programs = create_program_with_tiers()`. `create_program_with_tiers` would just call `create_program` to start